extensions/ql-vscode/src/common/commands.ts

```diff
@@ -0,0 +1,11 @@
+import { CommandManager } from "../packages/commands";
+
+export type ExtensionCommands = {
+  "codeQL.openVariantAnalysisLogs": (
+    variantAnalysisId: number,
+  ) => Promise<void>;
+};
+
+export type AllCommands = ExtensionCommands;
```

Comment:
This might be obvious, but what other types of commands besides variant analysis commands do we think we'd need? Should `ExtensionCommands` eventually have a less general name or do we intend to get rid of it?

Reply:
I think we could have at least the following command "groups":
- Variant analysis
- New database panel
- Old database panel
- Query history
- Local query
- Any other commands, like showing documentation

I think the last group would be the `ExtensionCommands`. I think we'd need to see which commands end up in that group to give it a better name.